db/seeds.rb

```diff
@@ -1,27 +1,18 @@
 class SeedData
+
   def users_list
     [
-      { firstname: "Chinedu", lastname: "Daniel",
-        email: "chinedu.daniel@andela.com", latitude: "",
-        longitude: "", password: "1234567890" },
-      { firstname: "Chinedu", lastname: "Daniel",
-        email: "chinedu.daniel@andela.com", latitude: "",
-        longitude: "", password: "1234567890" },
-      { firstname: "Chinedu", lastname: "Daniel",
-        email: "chinedu.daniel@andela.com", latitude: "",
-        longitude: "", password: "1234567890" },
-      { firstname: "Chinedu", lastname: "Daniel",
-        email: "chinedu.daniel@andela.com", latitude: "",
-        longitude: "", password: "1234567890" },
-      { firstname: "Chinedu", lastname: "Daniel",
-        email: "chinedu.daniel@andela.com", latitude: "",
-        longitude: "", password: "1234567890" }
+      { firstname: "Olaide", lastname: "Ojewale", email:"olaide.ojewale@andela.com", street_address: "55 Moleye Street", city: "Yaba", state: "Lagos", password:"1234567890", user_type: "taskee" },
+      { firstname: "Chinedu", lastname: "Daniel", email:"chinedu.daniel@andela.com",  street_address: "55 Moleye Street", city: "Yaba", state: "Lagos", password:"1234567890", user_type: "taskee" },
+      { firstname: "Temitope", lastname: "Amodu", email:"temitope.amodu@andela.com",  street_address: "2 Funso Street", city: "Yaba", state: "Lagos", password:"1234567890", user_type: "taskee" },
+      { firstname: "Ruth", lastname: "Chukwumam", email:"ruth.chukwumam@andela.com",  street_address: "44 Isaac John", city: "GRA", state: "Lagos", password:"1234567890", user_type: "taskee" },
+      { firstname: "Chinedu", lastname: "Dan", email:"chinedu.dan@andela.com",  street_address: "34, Adeyemo Alakija", city: "VI", state: "Lagos", password:"1234567890", user_type: "taskee" }
```

Comment:
Line is too long. [191/80]<br>Unnecessary spacing detected.<br>Space missing after colon.